pkg/hubble/parser/seven/http_test.go

```diff
@@ -423,3 +423,49 @@ func TestGetL7HTTPResponseTraceID(t *testing.T) {
 	_, ok = parser.traceContextCache.Get(requestID)
 	assert.False(t, ok, "request id should not be in the cache")
 }
+
+func TestFilterHeader(t *testing.T) {
```

Comment:
I think it might be better to test `decodeHTTP` and verify the behavior with multiple headers. This test can still exist, but I would like to see a test that has a flow with multiple headers, and then various redaction options.